assets/js/googlesitekit/datastore/site/developer-plugin.test.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * `core/site` data store, developer plugin tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	createTestRegistry,
+	muteFetch,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/site developer plugin state', () => {
+	const responseDeveloperPluginState = {
+		active: true,
+		installed: true,
+		activateURL: 'http://something.test',
+		installURL: 'http://something.test',
+		configureURL: 'http://something.test',
+	};
+
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'actions', () => {
+		it( 'does not require any params', () => {
+			expect( () => {
+				muteFetch( /^\/google-site-kit\/v1\/core\/site\/data\/developer-plugin/ );
+				registry.dispatch( STORE_NAME ).fetchGetDeveloperPluginState();
+			} ).not.toThrow();
+		} );
+
+		describe( 'receiveGetDeveloperPluginState', () => {
+			it( 'requires the response param', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).receiveGetDeveloperPluginState();
+				} ).toThrow( 'response is required.' );
+			} );
+
+			it( 'receives and sets developer plugin state', async () => {
+				await registry.dispatch( STORE_NAME ).receiveGetDeveloperPluginState( responseDeveloperPluginState );
```

Comment:
Receive actions are synchronous so the `await` is unnecessary here.
```suggestion
			it( 'receives and sets developer plugin state', () => {
				registry.dispatch( STORE_NAME ).receiveGetDeveloperPluginState( responseDeveloperPluginState );
```